--- a/defenvs/bananapi_bpi-r4_sdmmc_env
+++ b/defenvs/bananapi_bpi-r4_sdmmc_env
@@ -4,3 +4,3 @@
 bootconf_emmc=mt7988a-bananapi-bpi-r4-emmc
-bootconf_extra=mt7988a-bananapi-bpi-r4-leds
+bootconf_extra=mt7988a-bananapi-bpi-r4-leds#mt7988a-bananapi-bpi-r4-nopcie2
 bootdelay=0
--- a/defenvs/bananapi_bpi-r4_emmc_env
+++ b/defenvs/bananapi_bpi-r4_emmc_env
@@ -3,3 +3,3 @@
 bootconf_emmc=mt7988a-bananapi-bpi-r4-emmc
-bootconf_extra=mt7988a-bananapi-bpi-r4-leds
+bootconf_extra=mt7988a-bananapi-bpi-r4-leds#mt7988a-bananapi-bpi-r4-nopcie2
 bootdelay=0
--- a/defenvs/bananapi_bpi-r4-poe_sdmmc_env
+++ b/defenvs/bananapi_bpi-r4-poe_sdmmc_env
@@ -4,3 +4,3 @@
 bootconf_emmc=mt7988a-bananapi-bpi-r4-emmc
-bootconf_extra=mt7988a-bananapi-bpi-r4-leds
+bootconf_extra=mt7988a-bananapi-bpi-r4-leds#mt7988a-bananapi-bpi-r4-nopcie2
 bootdelay=0
--- a/defenvs/bananapi_bpi-r4-poe_emmc_env
+++ b/defenvs/bananapi_bpi-r4-poe_emmc_env
@@ -3,3 +3,3 @@
 bootconf_emmc=mt7988a-bananapi-bpi-r4-emmc
-bootconf_extra=mt7988a-bananapi-bpi-r4-leds
+bootconf_extra=mt7988a-bananapi-bpi-r4-leds#mt7988a-bananapi-bpi-r4-nopcie2
 bootdelay=0
--- a/defenvs/bananapi_bpi-r4_snand_env
+++ b/defenvs/bananapi_bpi-r4_snand_env
@@ -3,3 +3,3 @@
 bootconf=config-mt7988a-bananapi-bpi-r4
-bootconf_extra=mt7988a-bananapi-bpi-r4-spim-nand#mt7988a-bananapi-bpi-r4-emmc#mt7988a-bananapi-bpi-r4-leds
+bootconf_extra=mt7988a-bananapi-bpi-r4-spim-nand#mt7988a-bananapi-bpi-r4-emmc#mt7988a-bananapi-bpi-r4-leds#mt7988a-bananapi-bpi-r4-nopcie2
 bootcmd=mdio write mt7988 0 1f.24 c007 ; mdio write mt7988 1 1f.24 c007 ; mdio write mt7988 2 1f.24 c007 ; mdio write mt7988 3 1f.24 8007 ; if env exists nvme_boot ; then run boot_nvme ; else if pstore check ; then run boot_recovery ; else run boot_ubi ; fi ; fi
--- a/defenvs/bananapi_bpi-r4-poe_snand_env
+++ b/defenvs/bananapi_bpi-r4-poe_snand_env
@@ -3,3 +3,3 @@
 bootconf=config-mt7988a-bananapi-bpi-r4-poe
-bootconf_extra=mt7988a-bananapi-bpi-r4-emmc#mt7988a-bananapi-bpi-r4-leds
+bootconf_extra=mt7988a-bananapi-bpi-r4-emmc#mt7988a-bananapi-bpi-r4-leds#mt7988a-bananapi-bpi-r4-nopcie2
 bootcmd=mdio write mt7988 0 1f.24 c007 ; mdio write mt7988 1 1f.24 c007 ; mdio write mt7988 2 1f.24 c007 ; mdio write mt7988 3 1f.24 8007 ; if pstore check ; then run boot_recovery ; else run boot_ubi ; fi
